UPGRADE-4.1.md

```diff
@@ -0,0 +1,35 @@
+UPGRADE FROM 4.0 to 4.1
+=======================
+
+Form
+----
+
+ * Support for the `extended_type` attribute inside the `FormPass` is deprecated and will be dropped in Symfony 5.0.
+   Implement the `public static function extends(): string` method in your form type extension class instead.
```

Comment:
`extendsType()`? Same in `UPGRADE-5.0.md` and `src/Symfony/Component/Form/CHANGELOG.md`